### ISSUE TITLE
Derived form is write protected

### ISSUE THREAD

**Issue by r-schaefer** [dotnet/winforms#8882]
### Environment

Visual Studio 2022 - Community Edition

### .NET version

.Net 7.0

### Did this work in a previous version of Visual Studio and/or previous .NET release?

Yes: With Visual Studio 2019 and .Net Framework 4.6.2

### Issue description

A derived form cannot be edited with VS 2022:

See attached project:

- BaseForm.cs: is the base form class
- BaseMDIChild.cs: inherits from BaseForm.cs
- BaseMDIERPDataChild.cs: inherits from BaseMDIChild.cs

This form seems to be write protected. You cannot insert any control.

### Steps to reproduce

1. Open the project, compile it
2. Open the form BaseMDIERPDataChild.cs and try inserting a control

**Comment by r-schaefer** [comment]
[DXApplication58.zip](https://github.com/dotnet/winforms/files/11028736/DXApplication58.zip)

**Comment by merriemcgaw** [comment]
@Olina-Zhang can your team please try to repro this?

**Comment by Olina-Zhang** [comment]
> @Olina-Zhang can your team please try to repro this?

Yes, we can repro it, it was caused by Base Form's WindowsState with **Maximized** value set, repro with out of process designer in Visual Studio 16.11.25 to the latest, not in-process designer.

**Comment by r-schaefer** [comment]
Many thanks for your quick support. Resetting the WindowState to Normal fixed the issues:
- The form is no longer write protected
- All control are shown within the designer

**Comment by elachlan** [comment]
Is there a corresponding designer issue created?

**Comment by Olina-Zhang** [comment]
> WindowsState with **Maximized**

It is the designer issue: https://github.com/microsoft/winforms-designer/issues/5057.
